tests/integ/modin/frame/test_setitem.py

```diff
@@ -351,22 +351,29 @@ def func_insert_new_column(df, column):
 
         # convert to snow objects when testing for Snowpark pandas
         if isinstance(df, pd.DataFrame):
-            if isinstance(column, native_pd.Index):
+            if isinstance(column, native_pd.Index) and not isinstance(
+                column, native_pd.DatetimeIndex
+            ):
                 column = pd.Index(column)
-            elif isinstance(column, native_pd.Series):
+            if isinstance(column, native_pd.Series):
```

Comment:
why did you change the elif to if?

Reply:
Putting this back now, it was just a typo